projects/components/src/table/cells/data-parsers/table-cell-timestamp-parser.ts

```diff
@@ -0,0 +1,18 @@
+import { TableCellParser } from '../table-cell-parser';
+import { TableCellParserBase } from '../table-cell-parser-base';
+import { CoreTableCellParserType } from '../types/core-table-cell-parser-type';
+
+@TableCellParser({
+  type: CoreTableCellParserType.Timestamp
+})
+export class TableCellTimestampParser extends TableCellParserBase<CellData, CellData, CellData> {
+  public parseValue(cellData: CellData): CellData {
+    return cellData;
+  }
+
+  public parseFilterValue(cellData: CellData): CellData {
+    return this.parseValue(cellData);
```

Comment:
should this be in the base class?

Reply:
I see the the value in having a default implementation, but as mentioned in the other comment, would prefer to keep the method abstract.